python/cudf/cudf/utils/utils.py

```diff
@@ -37,6 +39,65 @@
 }
 
 
+# The test root is set by pytest to support situations where tests are run from
+# a source tree on a built version of cudf.
+NO_EXTERNAL_ONLY_APIS = os.getenv("NO_EXTERNAL_ONLY_APIS")
+_CUDF_TEST_ROOT = os.getenv("_CUDF_TEST_ROOT")
+
+if NO_EXTERNAL_ONLY_APIS in ("True", "1", "TRUE"):
+    _cudf_root = cudf.__file__.replace("/__init__.py", "")
```

Comment:
```suggestion
    _cudf_root = os.path.dirname(cudf.__file__)
```

Reply:
Not really sure why I didn't do that to begin with. Thanks.